tests/e2e/e2e_test.go

```diff
@@ -182,12 +183,23 @@ var _ = Describe("odoe2e", func() {
 
 	Describe("creating a component", func() {
 		Context("when application exists", func() {
+			It("should be able to create new imagestream and find it in catalog list", func() {
+				curProj = strings.TrimSuffix(runCmd("oc project -q"), "\n")
+				cmd := fmt.Sprintf("oc create -f https://raw.githubusercontent.com/bucharest-gold/centos7-s2i-nodejs/master/imagestreams/nodejs-centos7.json -n %s", curProj)
```

Comment:
it'd be nice to have this as a const / separate variable in-case it changes in the future. I'm referring to: https://raw.githubusercontent.com/bucharest-gold/centos7-s2i-nodejs/master/imagestreams/nodejs-centos7.json

Reply:
maybe:

```go
testNamespacedImage := "https://raw.githubusercontent.com/bucharest-gold/centos7-s2i-nodejs/master/imagestreams/nodejs-centos7.json"
```

Reply:
Ack